todo.txt

```diff
@@ -20,5 +23,12 @@
 		-Thus for consistency, if a function CAN apply to a file, 
 		 it should execute once before exiting	
 
-***BUG FUNCTION IDEAS***
+
+Other random thoughts:
+
+-- I think the fill in the blank is a good idea, but it seems like it is a little overused... a few lines here and there would be cool to see but with my testing, it gets a little obnoxious.
+
+-- The Bugged docstring may have an error but i am not compleltely sure yet.
```

Comment:
Currently buggify_docstring() is only designed to work if docstrings have triple quotes on multiple lines. For instance, this will work:
~~~~
def testfunction():
```
Some Text Here:
```
~~~~

However, if there are docstrings like this, the whole function won't work.
~~~~
def testfunction:
```Some Text Here```
~~~~
That is because there needs to be an even amount of lines with triple quotes. The list of lines with triple quotes gets split up into two lists -one is the beginning of the docstrings, one is the end.

I'm sure there is a better way to find docstrings. In any case, thanks for pointing it out!

Reply:
Sounds good, here is what happened on one of my example python programs that I ran Buggify on...

`def encrypt_message(msg):
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING
    #BUGGIFIED DOCSTRING`


Is this intentional or is this an error? I just wanted to clarify